src/documentation/programming-guide.md

```diff
@@ -917,7 +917,94 @@ While `ParDo` always produces a main output `PCollection` (as the return value f
 
 ## <a name="transforms-composite"></a>Composite Transforms
 
-> **Note:** This section is in progress ([BEAM-1452](https://issues.apache.org/jira/browse/BEAM-1452)).
+Transforms can have a nested structure, where a complex transform performs multiple simpler transforms (such as more than one `ParDo`, `Combine`, or `GroupByKey`). These transforms are called composite transforms. Nesting multiple transforms inside a single composite transform can make your code more modular and easier to understand.
+
+Many of Beam's pre-written transforms are composite transforms. See the API reference pages for a list of transforms:
```

Comment:
Maybe something like "The Beam SDK comes loaded with many composite transforms". I don't want to get too far into the weeds, but since there are only 5 _non_-composite transforms, maybe some phrasing can highlight the fact that we are providing in large part this big library of composites.